data/categories.txt

```diff
@@ -5041,39 +5041,269 @@ de:Leuchtturm
+vi:Đại lý Du lịch|chuyến du lịch
+zh-Hans:旅行社|游览
+zh-Hant:旅行社|遊覽
 
 shop-outdoor
 en:Outdoor Equipment|trekking|climbing|camping|shop
 ru:Снаряжение|магазин
+ar:معدات خارجية|ترحال|تسلق|تخييم|محل
+cs:Venkovní vybavení|trekking|lezení|kempování|obchod
+de:Outdoor-Ausrüstung|Trekking|Klettern|Camping|Geschäft
+es:Equipamiento|tienda
+fi:Ulkoiluvarusteet|vaellus|kiipeily|retkeily|kauppa
+fr:Matériel de loisirs de plein air|Randonnée|Escalade|Camping|Magasin
+hu:Kültéri felszerelés|trekking|mászás|kemping|üzlet|bolt
+id:Peralatan Outdoor|lintas alam|mendaki|kemah|toko
+it:Attrezzatura sportiva|trekking|arrampicata|campeggio|negozio
+ja:アウトドア用品|トレッキング|クライミング|店|買い物
+ko:야외 장비|트래킹|등반|캠핑|상점
+nb:Fritidsutstyr|vandring|klatring|camping|butikk
+pl:Sprzęt turystyczny|wędrowanie|wspinaczka|camping|sklep
+pt:Equipamentos esportivos|caminhada|escalada|camping|comércio|compras
+ro:Echipament de exterior|trekking|cățărare|campare|magazin
+sk:Outdoorové vybavenie|Treking|Lezenie|Kempy|obchod
+sv:Fritidsutrustning|vandring|klättring|camping|butik
+th:อุปกรณ์กลางแจ้ง|เดินป่า|ปีนเขา|ตั้งแคมป์|ร้าน
+tr:Dış Mekan Ekipmanları|yürüyüş|tırmanma|kamp|mağaza
+uk:Спорядження|магазин|крамниця
+vi:Thiết bị Ngoài trời|dã ngoại|leo|cắm trại|cửa hàng
+zh-Hans:室外设备|远足|攀岩|野营|商店
+zh-Hant:室外設備|遠足|攀巖|野營|商店
 
 shop-dry_cleaning
 en:Dry Cleaning|cleaning
 ru:Химчистка
+ar:غسيل جاف|غسيل
+cs:Chemické čištění|čistírna
+de:Chemische Reinigung|Reinigung
+es:Tintorería
+fi:Kuivapesula|siivous
+fr:Nettoyage à sec|Lavage
+hu:Ruhatisztító|tisztító
+id:Cuci Kering|kering
+it:Lavaggio a secco|lavanderia
+ja:クリーニング屋|クリーニング
+ko:드라이 클리닝|청소
+nb:Renseri|vaskeri
+pl:Pralnia chemiczna|czyszczenie
+pt:Lavagem a seco|lavanderia
+ro:Curățătorie chimică|curățătorie
+sk:Čistiareň|čistenie
+sv:Kemtvätt|tvätt
+th:ซักแห้ง|ทำความสะอาด
+tr:Kuru Temizleme|temizleme
+uk:Хімчистка
+vi:Giặt Sấy|giặt giũ
+zh-Hans:干洗|清洁
+zh-Hant:幹洗|清潔
 
 shop-tyres
 en:Tyre Shop|shop
 ru:Магазин шин|магазин
+ar:محل إطارات|محل
+cs:Obchod s pneumatikami|obchod
+de:Reifen-Shop|Geschäft
+es:Tienda de neumáticos|tienda
+fi:Rengasliike|kauppa
+fr:Magasin de pneus|Magasin
+hu:Gumi szaküzlet|üzlet|bolt
+id:Toko Ban|toko
+it:Gommista|negozio
+ja:タイヤ専門店|ショップ|買い物
+ko:타이어 상점|상점
+nb:Dekkforretning|butikk
+pl:Wulkanizacja|punkt
+pt:Borracheiro|comércio|compras
+ro:Magazin anvelope|magazin
+sk:Obchod s pneumatikami|obchod
+sv:Däckaffär|butik
+th:ร้านยาง|ร้านค้า
+tr:Tekerlekçi|dükkanı
+uk:Магазин шин|магазин|крамниця
+vi:Cửa hàng Lốp|cửa hàng
+zh-Hans:轮胎店|商店
+zh-Hant:輪胎店|商店
 
 amenity-car_wash
 en:Car Wash
 ru:Автомойка
+ar:مغسلة سيارات
+cs:Myčka aut
+de:Autowäsche
+es:Lavado de coches
+fi:Autopesula
+fr:Station de lavage
+hu:Autómosó
+id:Cuci Mobil
+it:Lavaggio auto
+ja:洗車
+ko:세차
+nb:Bilvask
+pl:Myjnia samochodowa
+pt:Lavagem de veículos
+ro:Spălătorie auto
+sk:Autoumývareň
+sv:Biltvätt
+th:ล้างรถ
+tr:Araba Yıkama
+uk:Автомийна
+vi:Rửa Xe
+zh-Hans:洗车
+zh-Hant:洗車
 
 amenity-veterinary
 en:Veterinary Doctor
 ru:Ветеринарная клиника
```

Comment:
Не нужно подстраивать отображение под поиск.

> On 25.3.2016, at 23.33, Ilya Zverev <notifications@github.com> wrote:
> 
> In data/categories.txt:
> 
> > +id:Cuci Mobil
> > +it:Lavaggio auto
> > +ja:洗車
> > +ko:세차
> > +nb:Bilvask
> > +pl:Myjnia samochodowa
> > +pt:Lavagem de veículos
> > +ro:Spălătorie auto
> > +sk:Autoumývareň
> > +sv:Biltvätt
> > +th:ล้างรถ
> > +tr:Araba Yıkama
> > +uk:Автомийна
> > +vi:Rửa Xe
> > +zh-Hans:洗车
> > +zh-Hant:洗車
> >  
> >  amenity-veterinary
> >  en:Veterinary Doctor
> >  ru:Ветеринарная клиника
> Я посчитал, что введя «ветеринар», человек найдёт «ветеринарную клинику» подстрокой.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>